data/na/swagger/1-0-5.yaml

```diff
@@ -410,6 +410,79 @@ paths:
           $ref: '#/responses/MethodNotAllowed'
         500:
           $ref: '#/responses/InternalServerError'
+  '/reports/commissions':
+    get:
+      summary: Commissions Report
+      description: |
+        This report provides details on fees applied to your custom commission payouts.
+
+      tags:
+        - Reports
+      operationId: getCommissions
+      parameters:
+        - name: from_date
+          in: query
+          description: "Report from date"
+          required: false
+          type: string
+          format: date
+        - name: to_date
+          in: query
+          description: "Report to date"
+          required: false
+          type: string
+          format: date
+        - name: report_days
+          in: query
+          description: "Number of days from today to generate the report for"
+          required: false
+          type: integer
+          format: integer
+        - name: filter_by
+          in: query
+          description: "Filter on payment status"
+          required: false
+          type: string
+          format: string
+        - name: filter_value
+          in: query
+          description: "Filer for payments with state - Scheduled, On Hold, Superseded, In Process, Approved, Declined, Pending Payment, Full Payment"
+          required: false
+          type: string
+          format: string
+        - name: start_row
+          in: query
+          description: "The first row number in the result set."
+          required: false
+          type: integer
+          format: integer
+        - name: end_row
+          in: query
+          description: "The last row number in the result set."
+          required: false
+          type: integer
+          format: integer
+        - name: version
+          in: query
+          description: "API version. Current version: "1.0""
```

Comment:
I think this line is causing compilation error. Maybe use single quote around the main value?